templates/checkout/details.html

```diff
@@ -50,25 +50,27 @@ <h3 class="checkout__review__tittle">{% trans "Order review" context "Order revi
               <p>{% trans "Shipment" context "Order review section" %}</p>
             </div>
             <div class="col-4">
-              <p class="float-right">{% if checkout.shipping_method %}
-                {% net checkout.shipping_method.get_total %}
-              {% else %}
-                &mdash;
-              {% endif %}</p>
+              <p class="float-right">
+                {% if checkout.shipping_method %}
+                  {{ checkout.shipping_method.get_total_price.gross|amount }}
+                {% else %}
+                  &mdash;
+                {% endif %}
+              </p>
```

Comment:
Please verify that the reformatting did not affect how it's rendered.

Reply:
Looking same